### PR TITLE
Add tests for DTOs in soul-admin.(#680)

### PR DESCRIPTION
**Pull request by jjnnzb**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-admin/src/test/java/org/dromara/soul/admin/dto/SpringMvcRegisterDTOTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.junit.jupiter.api.Assertions;
+
+/**
+ * Test case for {@link SpringMvcRegisterDTO}.
+ *
+ * @author Jiang Jining
+ */
+public final class SpringMvcRegisterDTOTest {
+    
+    private SpringMvcRegisterDTO springCloudRegisterDTO;
```
Comment (dengliming) [inline soul-admin/src/test/java/org/dromara/soul/admin/dto/SpringMvcRegisterDTOTest.java:31]: name

Comment (jjnnzb) [reply soul-admin/src/test/java/org/dromara/soul/admin/dto/SpringMvcRegisterDTOTest.java]: OK, modification already completed.

**Comment by dengliming** [comment]
Consider keep consistent with the testcases of `VO`

eg. extends `AbstractReflectGetterSetterTest`

**Comment by dengliming** [comment]
Thanks for your contribution. We don't need to test setter and getter method now.

**Pull request closed without merging**